witchcraft/witchcraft.go

```diff
@@ -679,6 +683,11 @@ func (s *Server) Start() (rErr error) {
 		}
 	}
 
+	// initialize log flushers
+	for _, logFlusher := range s.logFlushers {
```

Comment:
Why bother accumulating these and launching them in *Server? Couldn't this just `go periodicallyRotateLogFiles(ctx, logger, slsFilename)` in newDefaultLogOutputWriter?

Reply:
The rotation function stops if the context is cancelled and we only have a context in the main server function after all loggers are initialized.